site/Dance/index.php

```diff
@@ -181,102 +95,29 @@
     </div>
 
 
-    <div class="intro-text">
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit
+    <?php template("intro.php", array('introtext' => 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit
         tristique quam. Donec mollis auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui.
         Duis
         dictum in ligula a aliquet. Vestibulum consequat nisl a elementum tincidunt. Nunc nec egestas erat. Fusce
         efficitur,
         nibh nec convallis eleifend, est lorem tristique quam, sit amet aliquet elit lorem eu quam. Etiam pharetra
         aliquam
         orci, vitae blandit nisi maximus in. Integer ante eros, elementum vitae hendrerit eget, pharetra eget quam.
-        Donec
-        fringilla, nisi nec fermentum pellentesque, ante eros consectetur ante, at venenatis ex dui ut velit.</p>
-      <div class="intro-text-2">
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit
-          tristique quam. Donec mollis auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu
-          dui.
-          Duis
-          dictum in ligula a aliquet. Vestibulum consequat nisl a elementum tincidunt. Nunc nec egestas erat. Fusce
-          efficitur,
-          nibh nec convallis eleifend, est lorem tristique quam, sit amet aliquet elit lorem eu quam. Etiam pharetra
-          aliquam
-          orci, vitae blandit nisi maximus in. Integer ante eros, elementum vitae hendrerit eget, pharetra eget quam.
-          Donec
-          fringilla, nisi nec fermentum pellentesque, ante eros consectetur ante, at venenatis ex dui ut velit.</p>
-      </div>
-    </div>
+        Donec')) ?>
 
-    <div class="TBD-space">
-      <div class="TBD-text">TBD</div>
-    </div>
+<?php template("tbd.php", array('TBD' => 'TBD')) ?>
 
-    <div class="related-pages-start">
-      <h2>Related Pages</h2>
-      <div class="related-pages-items">
+   
 
-        <div class="related-pages-item">
-          <div class="related-pages-image">
-            <div class="related-pages-date">April<br>
-              <div class="related-pages-date-number">15</div>
-            </div>
-            <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle">
-          </div>
-          <div class="related-pages-wrapper">
-            <div class="related-pages-title">Dance Audtions</div>
-            <p>consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit tristique quam. Donec
-              mollis
-              auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui. </p>
-            <a>Learn More</a>
-          </div>
-        </div>
+<?php template("related.php", array('title' => 'Related Pages')) ?>
```

Comment:
I think you can merge `related.php` and `relateditems.php` unless there was a need to keep them separate?